givitsite/templates/feed.html

```diff
@@ -24,15 +24,23 @@ <h1>My Feed</h1>
         </tr>
     {% for item in founds %}
     {% if item.request_id == req.id %}
-            <tr>
-                <td>{{item.title}}</td>
-                <td>{{item.city}}</td>
-                <td><a href="{{item.url}}">link</a></td>
-                <td>{{item.phone_number}}</td>
-                <td><img src="{{item.picture}}" width='50px'/></td>
-                {% comment %} beyond-io/givit#22 {% endcomment %}
-                <td><button type='button' class='btn btn-primary'>match</button></td>
-            </tr>
+        <tr>
+            <td>{{item.title}}</td>
+            <td>{{item.city}}</td>
+            <td><a href="{{item.url}}">link</a></td>
+            <td>{{item.phone_number}}</td>
+            <td><img src="{{item.picture}}" width='50px'/></td>
+            <td>
+                <input type='hidden' name="id" value ={{item.id}} />
+                {% if item.match %}
```

Comment:
I think it could be clearer to just have the `if` block arounf the `disabled` keyword.